CHANGELOG.txt

```diff
@@ -214,6 +217,8 @@
         # In JavaScript :
             - The function 'creme.forms._toDualColumnMultiSelect' & 'creme.forms.toOrderedMultiSelect' have been removed ;
               the new component 'creme.widget.SelectOrInputWidget' is used instead.
+            - 'creme.component.Chosen' has been renamed as 'creme.form.Chosen' and moved into 'widget/form/chosen.js' file.
+            - 'creme.form.Chosen' is now deprecated : no longer used in media generation but still exist to ease migration.
```

Comment:
It's only one item : "creme.component.Chosen is now deprecated ; but it has been renamed ..."

Reply:
ok